build.gradle

```diff
@@ -310,7 +316,7 @@ configure(subprojects.findAll { it.name != 'util' && it.name != 'mongo-java-driv
 
 task docs(type: Javadoc) {
     source subprojects.grep({ it.name != 'util' }).collect {project -> project.sourceSets.main.allJava }
-    options = subprojects.first().javadoc.options
+//    options = subprojects.first().javadoc.options
```

Comment:
Do we have a ticket for handling the removed `Javadoc.setOptions(MinimalJavadocOptions)`?